### PR TITLE
Add Source Code Repo: CodeFever

### PR DESCRIPTION
**Pull request by shishirui**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,7 @@ Table of Contents
   * [chiselapp.com](http://chiselapp.com/) — Unlimited public and private Fossil repositories
   * [codebasehq.com](https://www.codebasehq.com/) — One free project with 100 MB space and 2 users
   * [Codeberg.org](https://codeberg.org/) - Unlimited public and private Git repos for free and open-source projects. Static website hosting with [Codeberg Pages](https://codeberg.page/).
+  * [CodeFever](https://codefever.cn) — Unlimited private repositories. Fully open source and lightweight desigin. Powered by PGYER Team.
```
Comment (ripienaar) [inline README.md:163]: please link to an english page so we can determine if it's really free, thank you

File: README.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ Table of Contents
   * [chiselapp.com](http://chiselapp.com/) — Unlimited public and private Fossil repositories
   * [codebasehq.com](https://www.codebasehq.com/) — One free project with 100 MB space and 2 users
   * [Codeberg.org](https://codeberg.org/) - Unlimited public and private Git repos for free and open-source projects. Static website hosting with [Codeberg Pages](https://codeberg.page/).
-  * [CodeFever](https://codefever.cn) — Unlimited private repositories. Fully open source and lightweight desigin. Powered by PGYER Team.
+  * [CodeFever](https://github.com/PGYER/codefever) — Unlimited private repositories. Fully open source and lightweight desigin. Powered by PGYER Team.
```
Comment (ripienaar) [inline README.md:163]: The github project does not tell me about the pricing plans or what is available for free in the feature sets provided by a Software As A Service offering.

Comment (shishirui) [reply README.md:163]: This is a completely open source project and is mentioned among his features as being completely free.

Comment (ripienaar) [reply README.md:163]: This list is not for self hosted open source software

**Pull request closed without merging**